draft-ietf-tls-tls13.md

```diff
@@ -3875,7 +3876,13 @@ and their allocation policies are below:
 - TLS Cipher Suite Registry: Values with the first byte in the range
   0-254 (decimal) are assigned via Specification Required {{RFC2434}}.
   Values with the first byte 255 (decimal) are reserved for Private
-  Use {{RFC2434}}. IANA [SHALL add/has added] a "Recommended" column
+  Use {{RFC2434}}.
+
+  IANA [SHALL add/has added] the cipher suites listed in {{cipher-suites}} to
+  the registry. The "Value" and "Description" columns are taken from the table.
+  The "DTLS-OK" column is marked as "Yes".
+
+  IANA [SHALL add/has added] a "Recommended" column
```

Comment:
This was another consequence of the new cipher suites. We are currently poised to mark every non-TLS-1.3 cipher as not recommended. Should I leave that as-is or transcribe the list from draft 14?